cypress/e2e/read-receipts/high-level.spec.ts

```diff
@@ -168,7 +168,23 @@ describe("Read receipts", () => {
             // The room is unread again
             assertUnread(room2, 1);
         });
-        it.skip("A receipt for the last unimportant event makes the room read, even if all are unimportant", () => {});
+        it("A receipt for the last unimportant event makes the room read, even if all are unimportant", () => {
+            // Display room 1
+            goTo(room1);
+
+            // The room 1 is read
+            assertRead(room1);
```

Comment:
We should do everything from here on on room2, so that we're not accidentally reading the events as they appear. (That's why we switch to room 1.)